src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,103 @@
+
+
+
+<figure class="w-figure  w-figure--center">
+  <img src="./image-cdn-requests.jpg" alt="Shows the requests/responses to/from an image CDN">
+</figure>
+
+
+Image CDNs are different from build-time image optimization scripts in that they create new versions of images as they are needed. As a result, they are generally better suited than build scripts for creating images that are heavily customized for each individual client.
+
+
+## Image URLs
+
+Image URLs used by image CDNs convey important information about an image and the transformations and optimizations that should be applied to it. URL formats will vary depending on image CDN, but at a high-level, they all have similar features.
+
+
+
+<figure class="w-figure  w-figure--center">
+  <img src="./image-cdn-url.jpg" alt="Example of a image CDN image URL">
+</figure>
+
+
+
+### Origin
+
+An image CDN can live on your own domain or the domain of your image CDN. Third-party image CDNs typically offer the option of using a custom domain (for example, yours) for a fee. Using your own domain makes it easier to switch image CDNs at a later date because no URL changes will be needed.
+
+Figure 2 uses the image CDN’s domain (“example-cdn.com”) with a personalized subdomain, rather than a custom domain.
+
+
+### Image
+
+A widely supported way of uploading images to an image CDN is to send them via an HTTP POST request to the image CDN’s API. However, manually uploading images is not always necessary in order to use an image CDN. Some image CDNs can be configured to automatically retrieve images from their existing locations when they are needed.
+
+This is often achieved by including the complete URL of the _existing image_ within the URL for the image generated by the image CDN. For example, you might see a URL that looks like this: `https://[my-site.example-cdn.com/https://flowers.com/daisy.jpg/quality=auto`. This URL would retrieve and optimize the image that exists at `https://flowers.com/daisy.jpg`.
+
+
+### Security key
+
+A security key prevents other people from creating new versions of your images. If this feature is enabled, each new version of an image requires a unique security key. If someone tries to change the parameters of the image URL but doesn’t provide a valid security key, they won’t be able to create a new version. Your image CDN will take care of the details of generating and tracking security keys for you.
+
+
+### Transformations
+
+Image CDNs offer tens, and in some cases hundreds, of different image transformations. These transformations are specified via the URL string and there are no restrictions on using multiple transformations at the same time. In the context of web performance, the most important image transformations are size, pixel density, format, and compression. These transformations are the reason why switching to an image CDN typically results in a significant reduction in image size.
```

Comment:
```suggestion
Image CDNs offer tens, and in some cases hundreds, of different image transformations. These transformations are specified via the URL string, and there are no restrictions on using multiple transformations at the same time. In the context of web performance, the most important image transformations are size, pixel density, format, and compression. These transformations are the reason why switching to an image CDN typically results in a significant reduction in image size.
```
You need a comma between two independent clauses joined by a coordinating conjunction.